kernel/Dockerfile.config

```diff
@@ -0,0 +1,24 @@
+FROM linuxkit/kernel-compile:1b396c221af673757703258159ddc8539843b02b@sha256:6b32d205bfc6407568324337b707d195d027328dbfec554428ea93e7b0a8299b AS kernel-build
+
+ARG KERNEL_VERSION
+ARG KERNEL_SERIES
+
```

Comment:
You don't need another `Dockerfile` to do this. Just copy the `.config` to a location like `/export/kernel_config` at the end of the normal Dockerfile and use a volume mount there to extract it.